src/video/cocoa/SDL_cocoamodes.m

```diff
@@ -262,7 +262,19 @@ static SDL_bool GetDisplayMode(_THIS, CGDisplayModeRef vidmode, SDL_bool vidmode
     mode->format = format;
     mode->w = width;
     mode->h = height;
-    mode->refresh_rate = refreshrate;
+
+    /* see GetDisplayModeRefreshRate() */
```

Comment:
Could `GetDisplayModeRefreshRate` be changed to output a numerator and denominator directly, instead of duplicating its functionality here?

Reply:
this isn't a real duplication: here's it always try to use the num/dem.
in the function, it does only if refresh_rate was 0 before.
i am not compiling on macosx, so i prefer no to change the whole behavior of this

Reply:
I can probably figure out the best approach after this gets merged.

Reply:
ok great ! ( or feel free to merge to something to my branch, but i'm not sure how to do this)